.github/workflows/dead-link-checker.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
     timeout-minutes: 30
     steps:
       - uses: actions/checkout@v2
-      - run: sudo npm install -g markdown-link-check@3.8.7
+      - run: sudo npm install -g markdown-link-check@3.9.3
```

Comment:
3.9.3 is buggy, the ignore rules are not respected, check https://github.com/apache/skywalking/pull/8466/files , the ignore rules are not respected by the checker

Reply:
But if we down to 3.8.7, it would mark our ppt link failed, see https://github.com/apache/dolphinscheduler-website/pull/657#issuecomment-1024872443